### PR TITLE
Wred ecn counters

### PR DESCRIPTION
**Pull request by AshokDaparthi**

### CODE REVIEW

File: inc/saiqueue.h [hunk 1]
```diff
@@ -229,28 +229,28 @@ typedef enum _sai_queue_stat_t
...
-    /** Get/set WRED green color dropped packets count [uint64_t] */
+    /** Get/set WRED green color dropped/marked packets count [uint64_t] */
     SAI_QUEUE_STAT_GREEN_DISCARD_DROPPED_PACKETS = 0x00000010,
 
-    /** Get/set WRED green color dropped bytes count [uint64_t] */
+    /** Get/set WRED green color dropped/marked bytes count [uint64_t] */
     SAI_QUEUE_STAT_GREEN_DISCARD_DROPPED_BYTES = 0x00000011,
 
-    /** Get/set WRED yellow color dropped packets count [uint64_t] */
+    /** Get/set WRED yellow color dropped/masked packets count [uint64_t] */
```
Comment (lguohan) [inline inc/saiqueue.h]: -> marked

File: inc/saiport.h [hunk 2]
```diff
@@ -1682,28 +1682,28 @@ typedef enum _sai_port_pool_stat_t
     /** SAI port stat if octets */
     SAI_PORT_POOL_STAT_IF_OCTETS,
 
-    /** Get/set WRED green packet count [uint64_t] */
+    /** Get/set WRED green dropped/marked packet count [uint64_t] */
     SAI_PORT_POOL_STAT_GREEN_DISCARD_DROPPED_PACKETS,
```
Comment (lguohan) [inline inc/saiport.h]: DISCARD -> WRED

Is it only for drop + mark, or drop only?

File: inc/saiqueue.h [hunk 3]
```diff
@@ -265,6 +265,30 @@ typedef enum _sai_queue_stat_t
...
+    SAI_QUEUE_STAT_GREEN_WRED_ECN_MARKED_BYTES = 0x0000001d,
+
+    /** Get/set WRED yellow color marked packets count [uint64_t] */
+    SAI_QUEUE_STAT_YELLOW_WRED_ECN_MARKED_PACKETS = 0x0000001e,
+
+    /** Get/set WRED yellow color marked bytes count [uint64_t] */
+    SAI_QUEUE_STAT_YELLOW_WRED_ECN_MARKED_BYTES = 0x0000001f,
+
+    /** Get/set WRED red color marked packets count [uint64_t] */
+    SAI_QUEUE_STAT_RED_WRED_ECN_MARKED_PACKETS = 0x0000001d,
```
Comment (kcudnik) [inline inc/saiqueue.h]: duplicated number with SAI_QUEUE_STAT_GREEN_WRED_ECN_MARKED_BYTES

Comment (kcudnik) [inline inc/saiqueue.h]: other entries are also duplicated

**Pull request merged**